index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+  <title>Jerry's Market Place</title>
+</head>
+<body>
+  <nav>
+    <ul>
+      <li>HOME</li>
+      <li>PRODUCTS</li>
+      <li>CONTACT</li>
+      <li>ABOUT</li>
+    </ul>
+  </nav>
+
+  <div>
+    <h1 class="bjmarket">Benjamin and Geraldo's Frozen Treat Market Place
+    </h1>
+      <div>
+        <img class="headshot" src="http://www.mikethefanboy.com/wp/wp-content/uploads/2013/08/ben-affleck-e1377289867576.jpg" alt="Ben Affleck's Smiling Face">
+        <img class="headshot" src="http://mmbiz.qpic.cn/mmbiz/jAq9wqyDucxdfPicdfVUclXALrGph9DmviafggJia1fwxK2Yc9w88ekABZhu3ELpHFlANl6lfmlR4PObxuqq69zJw/640?/1.jpg" alt="Geraldo Rivera's Smiling Face">
+      </div>
+  </div>
+
+  <article class="icecream">
+    <div>
+      <header>
+        <h1>Cherry Garcia</h1>
+      </header>
+      <section>
+        <img class="pint" src="http://www.benjerry.com/files/live/sites/systemsite/files/flavors/products/us/pint/cherry-garcia-detail.png" alt "Ben and Jerry's Cherry Garcia">
+
+        <div class="description">
+          <p>
+          Cherry ice cream with cherry chunks and pieces of fudge flakes.
+        </p>
+        </div>
+
+      <input type="text" name="icecream" id="cherry_garcia" value="Available" readonly>
```

Comment:
This is kind of a unique way to approach this. Is there a reason you used inputs instead of just using `<p>` or `<div>` tags?